Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -199,17 +208,33 @@ class TabLocationView: UIView {
         addGestureRecognizer(longPressRecognizer)
         addGestureRecognizer(tapRecognizer)
         
+        let buttonContentEdgeInsets = UIEdgeInsets(top: 6, left: 6, bottom: 6, right: 6)
+        
+        #if NO_REWARDS
         [readerModeButton, reloadButton, separatorLine, shieldsButton].forEach {
             $0.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
             $0.setContentHuggingPriority(.defaultHigh, for: .horizontal)
         }
         
-        // Make tappable area bigger than the icon.
-        shieldsButton.contentEdgeInsets = UIEdgeInsets(top: 6, left: 6, bottom: 6, right: 6)
+        let buttonsStackview = UIStackView(arrangedSubviews: [shieldsButton])
+        
+        shieldsButton.contentEdgeInsets = buttonContentEdgeInsets
+        #else
+        [readerModeButton, separatorLine, shieldsButton, rewardsButton].forEach {
```

Comment:
Where is the reload button?

Reply:
good find, might have missed it when rebasing latest changes